src/api/folderApi.js

```diff
@@ -1,6 +1,22 @@
 import api from '../utils/api';
 
-export const getFolderData = async () => {
+export const getSampleFolderData = async () => {
   const response = await api.get('/sample/folder');
   return response.data;
 };
+
+export const getUserFolderData = async () => {
+  const response = await api.get('/users/1/folders');
+  return response.data;
+};
+
+export const getUserLinkData = async activeFolderId => {
+  let url = '/users/1/links';
+
+  if (activeFolderId) {
+    url = `/users/1/links?folderId=${activeFolderId}`;
+  }
+
+  const response = await api.get(url);
```

Comment:
```suggestion
export const getUserLinkData = async activeFolderId => {
  let queryParams = {};

  if (activeFolderId) {
   queryParams['folderId'] = activeFolderId;
  }

  const response = await api.get('/users/1/links', { params: queryParams });
```

[중요도: 중간] axios를 쓰시고 계시기 때문에 2번째 인자로 params의 값을 넘기게 된다면 알아서 queryParams가 들어가게 되는 구조입니다! axios를 쓰시는 만큼 url을 직접 주입하는 것 보다 이런 방식이 나중에 여러 queryParams가 동적으로 들어갈 수 있으니 참고 부탁드립니다~

참고:
{ params :queryParams }라고 적어드렸었는데 이거는 배울 때 보기 쉽게하라고 이렇게 적었지만 저는 할때는 변수값을 params로 선언해 key와 value의 변수명을 같게 만들어 { params }처럼 선언할 떄가 더 많으니 참고부탁드립니다~

Reply:
오 알지 못했던 부분이었습니다. 너무 감사합니다.

commit : 20a45af871324dcff12a7aee0f7be7898120d855